vocab/credentials/v2/template.html

```diff
@@ -264,33 +264,36 @@ <h2>Diagram description</h2>
           On the left side of this column, there are four labeled ellipses, styled as Class. 
           These are, from top to bottom, "VerifiableCredential", "JsonSchemaCredential",
           "VerifiableCredentialGraph", and "VerifiablePresentation". 
-          There is also a single circle, not styled as Property or Class, labeled as "OR", and another box 
-          styled as a Property labeled as "verifiableCredential". 
+          There is also a small, unlabelled circle, as a crossing point for connector lines.
         </p>
         <p>
           The "VerifiableCredential" ellipse is connected through a connecting line
```

Comment:
Note that this deletes line 270, which is replaced in the suggestion below, to work around GitHub change request limitations.
```suggestion
```